spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -2800,25 +2796,25 @@ always has a current transaction within which its work is done.
 JTS or some other transaction monitor facility may be used to combine a
 session’s transaction with transactions on other resources (databases,
 other Jakarta Messaging Sessions, etc.). Since Java distributed transactions are
-controlled via the JTA transaction demarcation API, use of the session’s
+controlled via the Jakarta Transactions transaction demarcation API, use of the session’s
 commit and rollback methods in this context throws a Jakarta Messaging
 TransactionInProgressException.
 
 ==== Distributed transactions
 
 Jakarta Messaging does not require that a provider support distributed transactions;
 however, it does define that if a provider supplies this support it
-should be done via the JTA XAResource API.
+should be done via the Jakarta Transactions XAResource API.
```

Comment:
XAResource API is still under JDK and not part of Jakarta Transactions.

Reply:
So what do you suggest? Revert this back to JTA XAResource API?

Reply:
Change yo Java XA Resource API as suggested.